lib/linter/timing.js

```diff
@@ -44,6 +44,21 @@ const enabled = !!process.env.TIMING;
 const HEADERS = ["Rule", "Time (ms)", "Relative"];
 const ALIGN = [alignLeft, alignRight, alignRight];
 
+/**
+ * Decide how many rules to show in the output list.
+ * @returns {number} the number of rules to show
+ */
+function getListSize() {
+    if (process.env.TIMING.toLowerCase() === "all") {
```

Comment:
Can we check again if `process.env.TIMING` is set, just to be sure that this cannot crash in some scenarios when ESLint is used through the API.

Something like this at the beginning of `getListSize`, to preserve the current behavior:

```js
if (typeof process.env.TIMING !== "string") {
    return MINIMUM_SIZE;
}
```

Reply:
Good point, fixed.